.pipelines/templates/template-deploy-int-env.yml

```diff
@@ -0,0 +1,105 @@
+parameters:
+  workingDirectory: ""
+steps:
+  - script: |
+      set -eu
+      cd ${{ parameters.workingDirectory }}
+
+      . secrets/env
+
+      MDM_CERT="$(cat secrets/mdm-cert.pem)"
+      MDM_KEY="$(cat secrets/mdm-key.pem)"
+      SSH_PUBLIC_KEY="${{ parameters.sshPublicKey }}"
+
+      echo "########## Creating RG $RESOURCEGROUP in $LOCATION ##########"
+      az group create -g "$RESOURCEGROUP" -l "$LOCATION" >/dev/null
+      echo "## Done"
+      
+      RP_SPN_ID=$(az group deployment show -n rp-nsg -g $RESOURCEGROUP --query properties.outputs.rpServicePrincipalId.value -o tsv)
+
+      echo "########## Deploying rp-production in RG $RESOURCEGROUP ##########"
+      echo "### domainName            = $DOMAIN_NAME.$PARENT_DOMAIN_NAME"
+      echo "### keyvaultPrefix        = $KEYVAULT_PREFIX"
+      echo "### rpServicePrincipalId  = $RP_SPN_ID"
+      az group deployment create \
+        -g "$RESOURCEGROUP" \
+        -n rp-int \
+        --template-file deploy/rp-production.json \
+        --parameters "databaseAccountName=$COSMOSDB_ACCOUNT" \
+        --parameters "domainName=$DOMAIN_NAME.$PARENT_DOMAIN_NAME" \
+        --parameters "fpServicePrincipalId=$AZURE_FP_CLIENT_ID" \
+        --parameters "keyvaultPrefix=$KEYVAULT_PREFIX" \
+        --parameters "mdmCertificate=$MDM_CERT" \
+        --parameters "mdmFrontendUrl=$MDM_FRONTEND" \
+        --parameters "mdmMetricNamespace=$MDM_NAMESPACE" \
+        --parameters "mdmMonitoringAccount=$MDM_ACCOUNT" \
+        --parameters "mdmPrivateKey=$MDM_KEY" \
+        --parameters "pullSecret=$PULL_SECRET" \
+        --parameters "rpImage=$RP_IMAGE" \
+        --parameters "rpImageAuth=$RP_IMAGE_AUTH"\
+        --parameters "rpMode=int" \
+        --parameters "sshPublicKey=$SSH_PUBLIC_KEY" \
+        --parameters "vmssDomainNameLabel=aro-int" \
+        --parameters "vmssName=int" \
+        --parameters "rpServicePrincipalId=$RP_SPN_ID" >/dev/null
+      ARM_TEMPLATE_STATE=$(az group deployment show -n rp-int -g $RESOURCEGROUP --query properties.provisioningState -o tsv)
+      if [[ $ARM_TEMPLATE_STATE == "Failed" ]]; then
+        echo "##[error] Error deploying env-development $(az group deployment show -n rp-int -g $RESOURCEGROUP --query properties.error.details -o tsv)"
+        exit 1
+      fi
+      echo "## Done"
+
+      echo "########## Adding access polices to -svc keyvault to be able to upload secrets ##########"
+      az keyvault set-policy -n "$KEYVAULT_PREFIX-svc" --spn $DEVOPS_CLIENT_ID -g $RESOURCEGROUP --secret-permissions get set --certificate-permissions get import
+
+      echo "########## Uploading required runtime secrets to -svc keyvault ##########"
+      if $(az keyvault secret show  --vault-name "$KEYVAULT_PREFIX-svc" -n encryption-key >/dev/null) ; then
+        echo "## Encryption key found skipping uploading"
```

Comment:
Can we add some punctuation to these messages? I've commented at the places I noticed this.
> Encryption key found; skipping uploading

(It doesn't have to be a semicolon.)